README.md

```diff
@@ -26,15 +26,21 @@ Run tests
 docker-compose run web make test
 ```
 
-You can drop into a bash shell in the container with 
+You can drop into a bash shell in the container with
 
 ```sh
 docker-compose run web bash
 ```
```

Comment:
oh wow I forgot we had docker instructions here, are these still valid? should we remove? (maybe separate PR)

Reply:
> are these still valid? should we remove?

No idea, on either point 😅 The `docker-compose.yml` looks relatively straight-forward, so if it was working at some point, I assume (with low confidence) that it probably wouldn't be impacted by these changes.